app/views/project_groups/index.haml

```diff
@@ -29,7 +29,11 @@
       %tbody
         - @project_groups.each do |group|
           %tr
-            %td= link_to group.name, edit_project_group_path(group)
+            %td
+              = link_to group.name, edit_project_group_path(group)
+              - if group.used_for_cas_sync?
```

Comment:
This will call `GrdaWarehouse::Config.get` on each iteration, but I think it's ok (i.e. not worth pulling the cas group ID out into a variable) because the config is cached. Is that right?